release-to-candidate/action.yaml

```diff
@@ -75,12 +81,15 @@ runs:
         # https://bugs.launchpad.net/snapcraft/+bug/1885150
         yq -i '.architectures |= [{"build-on": env(arch)}]' "$yaml_path"
 
-        cd "$project_root" || exit
+        cd "$project_root" && git init || exit
```

Comment:
Could this `git init` not fail if `project_root` is unset, because there would already be a git repo initialised?

Reply:
Hopefully not, because it'll just reinitialize then.

```bash
 soumyaxubuntu  soumyas-gf63-thin-11uc  ../Project/alacritty  master  git init
Reinitialized existing Git repository in /home/soumyaxubuntu/Downloads/Project/alacritty/.git/
```